src/app/shared/users/user.services.ts

```diff
@@ -1,11 +1,12 @@
+import { userMock } from './user.mocked';
 import { Injectable } from '@angular/core';
 import { User } from './user.interface';
 
 @Injectable({ providedIn: 'root' })
 export class UserService {
-    user: User = {
-        id: 1,
-        firstName: 'Jack',
-        lastName: 'Developer'
-    };
+    user: User = userMock;
```

Comment:
It is not a good idea to mix mock/test data.
You can have a `DEFAULT_USER` constant. export it and use in tests, but not in opposite order...

Reply:
done